app/pb_kits/playbook/pb_home_address_street/_home_address_street.jsx

```diff
@@ -15,6 +17,12 @@ const dot = (houseStyle) =>
   }
 }
 
+function titleize_address_cont(address_cont) {
+  if (!!address_cont) {
+   return titleize(address_cont);
+  }
+}
```

Comment:
Please, follow the javascript naming convention (lower-first camel case). Also, define this function as a constant:

```suggestion
const titleizeAddessCont = (addressCont) => addressCont ? titleize(addressCont) : null
```

Reply:
These changes have been added